api/envoy/api/v2/core/grpc_service.proto

```diff
@@ -86,6 +86,27 @@ message GrpcService {
         }
       }
 
+      // [#next-free-field: 10]
+      message StsService {
+        string token_exchange_service_uri = 1 [(validate.rules).string = {uri: true}];
```

Comment:
Can you add comments to all these fields?

Reply:
I have added comments to all these fields. Thanks.